call-home/src/common/constants.rs

```diff
@@ -6,7 +6,10 @@ use utils::version_info;
 
 /// PRODUCT is the name of the project for which this call-home component is deployed.
 pub fn product() -> String {
-    ::constants::product_train()
+    env::var("PRODUCT_NAME")
+        .ok()
+        .filter(|v| !v.is_empty())
+        .unwrap_or(::constants::product_train())
```

Comment:
Is this working well for both - when env var is not set and also when it is set to blank "" ?

Reply:
Yeah, it is. `env::var()` returns an `Result::Err()` for unset env, but an `Ok()` for empty env. That's why there's a `filter(|v| !v.is_empty())` to cut that case out as a `None()`.

I'll add a unit test, as I had the same concern.

Reply:
Added a unit test.